ledger/encoded/recordsV6.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
```

Comment:
this is all catchpoint-related, so why not in catchpoint package? Plus `encoded` name is kind of weird

Reply:
there is no `catchpoint` package. I thought about creating one, but didn't want the refactor to get continue spiraling.